fuseimg/requirements.txt

```diff
@@ -3,6 +3,6 @@
 scikit-image>=0.17.2
 torchvision>=0.8.1
 SimpleITK>=1.2.0
-opencv-python<=4.3.0.36 
+opencv-python>4.3.0.38 
```

Comment:
@mosheraboh, can you explain why `>4.3.0.38` is used, i.e. an exclusive condition and that particular version?
I would rather expect `>=4.2.0.32`, directly associated to [CVE-2019-5064](https://github.com/advisories/GHSA-q799-q27x-vp7w).

Reply:
We had an issue with 4.3.0.37 & 4.3.0.38 (in fuse1) and I can't reproduce them anymore.
I will follow your suggestion then and revisit if we will have a similar issue in the future.